assets/js/blocks/product-categories/index.js

```diff
@@ -31,6 +31,7 @@ registerBlockType( 'woocommerce/product-categories', {
 	example: {
 		attributes: {
 			hasCount: true,
+			hasImage: true,
```

Comment:
With this change, the preview looks misaligned vertically:

![imatge](https://user-images.githubusercontent.com/3616980/74644459-9a9a0f80-5176-11ea-8266-3caec3805405.png)

Should we set `hasImage` to false considering it's the default? Otherwise, if we want to set it to `true`, is there any way to center it vertically?